schema/validator.go

```diff
@@ -105,13 +137,26 @@ func validateManifestDescendants(r io.Reader) error {
 		return errors.Wrap(err, "manifest format mismatch")
 	}
 
+	if header.MediaType != string(v1.MediaTypeImageManifest) {
+		if strict {
+			return fmt.Errorf("error: manifest has an unknown media type: %s\n", header.MediaType)
+		}
+		fmt.Printf("warning: manifest has an unknown media type: %s\n", header.MediaType)
```

Comment:
Updated in 
https://github.com/opencontainers/image-spec/pull/452/commits/6934d41165d1441fe3bfd67dbd87e827395ce8d1
https://github.com/opencontainers/image-spec/pull/452/commits/3da0cd8feb09f373d0a8e99a5c430fc49f23667a
https://github.com/opencontainers/image-spec/pull/452/commits/a2d0b365daafe02246c57221f181277e988ce799

I code it as,
```go
  msg := fmt.Sprintf(...)
  if strict {
          return fmt.Errorf("error: %s", msg)
  }   
  fmt.Printf("warning: %s\n", msg)
```

I honestly like `fmt.Sprintf` to generate message, because it may be error or warning type.
`fmt.Errorf` and `fmt.Printf` can distinguish returning or omitting behavior.